dojo/templatetags/authorization_tags.py

```diff
@@ -1,11 +1,16 @@
 from django import template
 from crum import get_current_user
 from dojo.authorization.roles_permissions import Permissions
-from dojo.authorization.authorization import user_has_permission
+from dojo.authorization.authorization import user_has_global_permission, user_has_permission
 
 register = template.Library()
 
 
 @register.filter
 def has_object_permission(obj, permission):
     return user_has_permission(get_current_user(), obj, Permissions[permission])
+
+
+@register.filter
+def has_permission(permission):
```

Comment:
```suggestion
def has_global_permission(permission):
```

Then it fits to the other checks invented with this PR.

Reply:
Yeah maybe. I chose this one as I think the other one should be called has_object_permission. But I will change it and we'll see later if we want to do a big revamp or not.

Reply:
With the authorization for configuration we will get a `has_configuration_permission` as well.Then we will have a nice triplet. I agree, the object based permission checks should have `object` as part of their names. I would like to do it after the legacy authorization has been removed.